src/config/testnet.ts

```diff
@@ -0,0 +1,19 @@
+import { Config } from "~/config/config";
+import { createKitWalletUrls } from '~/config/kitWallet';
+import { createNearBlocksUrls } from '~/config/nearBlocks';
+
+export const testnet: Config = {
+  networkId: "testnet",
+  urls: {
+    rpc: "https://rpc.testnet.near.org",
+    myNearWallet: "https://testnet.mynearwallet.com",
+    kitWallet: createKitWalletUrls("https://testnet-api.kitwallet.app"),
+    nearBlocks: createNearBlocksUrls("https://testnet.nearblocks.io"),
+  },
+  accounts: {
+    multisigFactory: "multisignature.testnet",
+    lockupFactory: "lockup.devnet", // TODO create a real one
```

Comment:
Why create a real one? If i remember correctly, https://testnet.nearblocks.io/address/lockup.devnet is fully working on testnet